pystac/html/Catalog.jinja2

```diff
@@ -0,0 +1,26 @@
+{% import 'Macros.jinja2' as macros %}
+
+<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-mod-trusted jp-OutputArea-output">
+    {{ macros.square() }}
+    <div>
+        <details style="margin-left: 48px;">
+            <summary style="margin-bottom: 20px;">
+                <h3 style="margin-bottom: 0px; display: inline;">{{catalog_type}}: {{catalog.id}}</h3>
```

Comment:
```suggestion
                <h3 style="margin-bottom: 0px; display: inline;">{{catalog.__class__.__name__}}: {{catalog.id}}</h3>
```

Should we maybe get the class name directly from the instance here? This would avoid the necessity for the `catalog_type` argument (see my other comment), and would also mean that classes inheriting from `Catalog` or `Collection` in downstream libraries (e.g. [`pystac_client.CollectionClient`](https://github.com/stac-utils/pystac-client/blob/47812c6f59e8f81956e62f2149d4becc34e7aae0/pystac_client/collection_client.py#L10)) would be represented by their true class name.